README.md

```diff
@@ -0,0 +1,254 @@
+# Coding Agent Context CLI
+
+A CLI tool for managing context files for coding agents. It helps you organize prompts, memories (reusable context), and bootstrap scripts that can be assembled into a single context file for AI coding agents.
+
+It's aimed at coding agents with a simple interface for managing task-specific context and reusable knowledge.
+
+## Installation
```

Comment:
Add an example like this instead of the "from source" option:
```bash
sudo curl -fsL  -o /usr/local/bin/jira https://github.com/kitproj/jira-cli/releases/download/v0.0.8/jira_v0.0.8_linux_arm64
sudo chmod +x /usr/local/bin/jira
```